.github/CODEOWNERS

```diff
@@ -218,7 +218,7 @@ system/default_ad_api_helpers/automatic_pose_initializer/** isamu.takagi@tier4.j
 system/diagnostic_graph_aggregator/** isamu.takagi@tier4.jp
 system/diagnostic_graph_utils/** isamu.takagi@tier4.jp
 system/autoware_dummy_diag_publisher/** fumihito.ito@tier4.jp tetsuhiro.kawaguchi@tier4.jp
-system/dummy_infrastructure/** ryohsuke.mitsudome@tier4.jp
```

Comment:
You don't have to modify this file as it would be automatically updated from package.xml file.